### ISSUE TITLE
 future data when labeling

### ISSUE THREAD

**Issue by alvinonmars** [bradleyboyuyang/ML-HFT#4]
I think future data was introduced during training. When labeling, the training set data looked into the future 15 minutes; in this case, the test set would have a 15-minute gap with the training set, but in fact there is not.
